Modules/_zstd/_zstdmodule.c

```diff
@@ -28,43 +28,44 @@ set_zstd_error(const _zstd_state* const state,
     switch (type)
     {
     case ERR_DECOMPRESS:
-        msg = "Unable to decompress zstd data: %s";
+        msg = "decompress zstd data";
         break;
     case ERR_COMPRESS:
-        msg = "Unable to compress zstd data: %s";
+        msg = "compress zstd data";
         break;
     case ERR_SET_PLEDGED_INPUT_SIZE:
-        msg = "Unable to set pledged uncompressed content size: %s";
+        msg = "set pledged uncompressed content size";
```

Comment:
I think `ERR_SET_PLEDGED_INPUT_SIZE` is not use anywhere in the code base, maybe remove it?